monitor.py

```diff
@@ -901,6 +904,30 @@ def delete(self, collection):
 ReaperMonitor.REGISTRY.append(CollectionReaper)
 
 
+class MeasurementReaper(ReaperMonitor):
+    """Remove measurements that are not the most recent"""
+    MODEL_CLASS = Measurement
+    MEASUREMENT_REAPER_ENABLED = 'MeasurementReaper.enabled'
```

Comment:
This looks like an old name for the sitewide setting that you ended up not using.

Reply:
Good catch, thanks! Decided to change where it was defined.